save-deploy/README.md

```diff
@@ -24,7 +24,7 @@ Deployment is performed on server via docker swarm or locally via docker-compose
 * Secrets should be added to the swarm as well as to `$HOME/secrets` file.
 * If custom SSL certificates are used, they should be installed on the server and added into JDK's truststore inside images. See section below for details.
 * Loki logging driver should be added to docker installation: [instruction](https://grafana.com/docs/loki/latest/clients/docker-driver/#installing)
-* Pull new changes to the server and run `./gradlew -Psave.profile=prod deployDockerStack`.
+* Pull new changes to the server and run `./gradlew -Psave.profile=prod -PdockerNetwork=save_default deployDockerStack`.
```

Comment:
is it a suggestion or we should always provide -PdockerNetwork on prod deploying?

Reply:
We should always provide it, because network name is built from stack name (created with `docker stack deploy` command) and can't be deduced based on compose file location, as for docker-compose